keyboards/0xcb/splaytoraid/keymaps/pi/layers.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+// layer enum moved here so it can also be used by other files
```

Comment:
These aren't used anywhere else, and should be moved to the keymap (and this file removed)